tests/integration/test_analysisd/test_signature_id/data/test_cases/cases_invalid_signature_id.yaml

```diff
@@ -0,0 +1,13 @@
+- name: Empty Signature ID - wazuh restarts - rule ignored
+  description: Check that if_sid is empty, a rule will be ignored
+  configuration_parameters:
+  metadata:
+    rules_file: empty_signature_id_rules.xml
+    is_empty: true
+
+- name: Invalid Signature ID - wazuh restarts - rule ignored
```

Comment:
```suggestion
- name: Invalid signature ID - Rule ignored
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3649/commits/cf8e8008bb6aaf5eaf95b75a4c64f3757fba2713